apis/ui/v1alpha1/pgbouncer_pools_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright AppsCode Inc. and Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ResourceKindPgBouncerPools = "PgBouncerPools"
+	ResourcePgBouncerPools     = "pgbouncerpools"
+	ResourcePgBouncerPoolss    = "pgbouncerpools"
+)
+
+type PgBouncerPoolsSpec struct {
+	Pools []PBPools `json:"pools"`
+}
+
+type PBPools struct {
+	// PodName represents the corresponding pod for the pool
+	PodName string `json:"podName"`
+	// Database represents the database of the pool
+	Database string `json:"database"`
+	// User represents the user of the pool
+	User string `json:"user"`
+	// ClientConnectionsActive represents client connections that are either linked to server connections
+	// or are idle with no queries waiting to be processed
+	ClientConnectionsActive *int32 `json:"clientConnectionsActive"`
+	// ClientConnectionsWaiting represents client connections that have sent queries
+	// but have not yet got a server connection
+	ClientConnectionsWaiting *int32 `json:"clientConnectionsWaiting"`
+	// ClientsActiveCancelReq represents client connections that have forwarded query cancellations to the server
+	// and are waiting for the server response
+	ClientsActiveCancelReq *int32 `json:"clientsActiveCancelReq"`
+	// ClientsWaitingCancelReq represents client connections that have not forwarded query cancellations to the server yet.
```

Comment:
Description can be `...that are waiting to forward query cancellation request to the server`. WDYT?